components/base/Typography.tsx

```diff
@@ -28,7 +28,7 @@ export const headingStyles = {
   h1: css`
     font-size: 42px;
     line-height: 52px;
-    margin-top: 0;
+    margin-top: 26px;
```

Comment:
Added some top margin to this as well since I think we were under the assumption that `h1`s would only appear at the top of the page.

Reply:
Could you elaborate on this? If `h1`s are (generally) reserved for the top of the page, wouldn't we want them to not have any top margin so they align with the "On This Page" better?

Reply:
Yeah we have some pages with `h1`s in the middle of the page, but if we want we could make the top margin 0 for the first `h1` on the page to make it line up with the "On This Page" stuff. I kinda like it with the additional spacing on the top to match the whitespace below the heading, but I could go either way.

Reply:
Fair enough! Thanks for considering it; I'll defer to your judgement